salt/cache/redis_cache.py

```diff
@@ -493,6 +496,11 @@ def list_(bank):
         return []
     return list(bank_keys)
 
+def ls(bank):
```

Comment:
To avoid redundant code, I'd recommend renaming the above function `list_` to `ls`, while the `__func_alias__` dunder should be `{'list': 'ls'}`.